### PR TITLE
nodejs: update to 7.7.1

### PR DESCRIPTION
**Pull request by Sir-Boops**

### CODE REVIEW

File: srcpkgs/nodejs/template [hunk 1]
```diff
@@ -11,9 +11,11 @@ depends="python"
 short_desc="Evented I/O for V8 javascript"
 maintainer="Enno Boland <gottox@voidlinux.eu>"
 license="MIT"
-homepage="http://nodejs.org/"
+homepage="https://nodejs.org/"
 distfiles="${homepage}/dist/v${version}/node-v${version}.tar.gz"
-checksum=809e80265e332fe1a8268e5a73eb219c356810fe86c69fd2d931c52e07211970
+checksum=9e87ec5420d558ca9651d13b10dd4a1be954fea0fc7a909016a1cc4aedfa651c
+# TODO: Fix cross compiling
+nocross=yes
```
Comment (Vaelatern) [inline srcpkgs/nodejs/template]: `nocross="https://build.voidlinux.eu/builders/armv7l_builder/builds/28955/steps/shell_3/logs/stdio"`

Comment (Sir-Boops) [inline srcpkgs/nodejs/template]: Added, just seems weird to keep since it 404's.

Comment (Vaelatern) [inline srcpkgs/nodejs/template]: Can you generate a travis run and link to that instead? (unset nocross, push, set nocross to a link to a failed run)

Comment (Sir-Boops) [inline srcpkgs/nodejs/template]: Added the new link

**Pull request merged**